examples/sprites/src/lib.rs

```diff
@@ -27,34 +29,35 @@ pub fn translate(p: &Point, v: Vector) -> Point {
 
 // An entity in our imaginary world, which occupies a position in space
 // and which can move about over time.
-#[derive(Debug, Clone)]
+#[derive(Debug)]
 pub struct Sprite {
-    current_position: Point,
+    current_position: RwLock<Point>,
```

Comment:
For educational purposes, adding a short comment here about "All interfaces must be threadsafe, so we need to wrap this in a lock" may help new users for the correct mental model of how we expect threadsafety to work.